mat-process-api.Tests/V1/Helper/MatProcessDataHelper.cs

```diff
@@ -17,17 +18,38 @@ public static MatProcessData CreateProcessDataObject()
 
             return new MatProcessData
             {
+                Id = faker.Random.Guid().ToString(),
+                ProcessType = new ProcessType()
+                {
+                    value = faker.Random.Int(),
+                    name = faker.Random.Word()
+                },
                 DateCreated = faker.Date.Recent(),
                 DateLastModified = faker.Date.Recent(),
-                ProcessData = {},
-                ProcessDataSchemaVersion = faker.Random.Int(0,10),
                 DateCompleted = faker.Date.Recent(),
-                Id = faker.Random.Word(),
-                PostProcessData = {},
-                PreProcessData = {},
+                ProcessDataAvailable = false,
+                ProcessDataSchemaVersion = faker.Random.Int(0,10),
                 ProcessStage = faker.Random.Word(),
```

Comment:
The way it has been used doesn not matter in the context of ProcessStage. All that's being checked is whether the inserted docs data matches the retrieved one's. Therefore I can change this helper method to generate the "Not Completed" value for ProcessStage rather than the random value. It's better this way for consistency reasons, because the method in the edit is the method for generating **initial** document as can be seen from from other properties it generates.